routing_algorithms/routing_base.hpp

```diff
@@ -261,7 +261,7 @@ template <class DataFacadeT, class Derived> class BasicRoutingInterface
                                                    TurnInstruction::NoTurn, 0, travel_mode);
                     }
                     unpacked_path.back().turn_instruction = turn_instruction;
-                    unpacked_path.back().segment_duration = ed.distance;
```

Comment:
Aha! That’s was what I overlooked. Thank you. I will fix that ICE back to Paris.